### PR TITLE
[v2] Update dependencies

### PR DESCRIPTION
**Pull request by enahum**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "extends": [
+    "plugin:mattermost/react",
```
Comment (avinashlng1080) [inline .eslintrc.json:3]: The MM plugin should be last; if not their lint settings are not applied.

Comment (enahum) [reply .eslintrc.json:3]: not really.. having it last causes a million problems.

Comment (enahum) [reply .eslintrc.json:3]: can you point out why it should be last? btw it is first on v1

**Review by avinashlng1080 — APPROVED** [review]
LGTM.

File: .eslintrc.json (hunk 1 above)
Comment (avinashlng1080) [reply .eslintrc.json:3]: Alright.  We can discuss it during our meetings.

**Pull request merged**